components/automate-cli/cmd/chef-automate/summary.go

```diff
@@ -0,0 +1,442 @@
+	}
+	return nil
+}
+
+func (ss *Summary) executeCommandForArrayofIPs(sshUtil SSHUtil, remoteIps []string, script, serviceName, serviceType string) error {
+	getConfigJsonString := convTfvarToJson(a2haHabitatAutoTfvars)
+	for i := 0; i < len(remoteIps); i++ {
+		sshUtil.getSSHConfig().hostIP = remoteIps[i]
+		if serviceType == "FE" {
+			status := ss.getFEStatus(sshUtil, script, remoteIps[i], serviceName)
+			ss.feStatus = append(ss.feStatus, status)
+		}
+		if serviceType == "BE" {
+			authToken, err := ss.readA2haHabitatAutoTfvarsAuthToken(getConfigJsonString)
+			if err != nil {
+				return err
+			}
+			status, err := ss.getBEStatus(sshUtil, remoteIps[i], authToken, serviceName)
+			if err != nil {
+				return err
+			}
+			ss.beStatus = append(ss.beStatus, status)
+		}
+
+	}
+	return nil
+}
+
+func (ss *Summary) readA2haHabitatAutoTfvarsAuthToken(getConfigJsonString string) (string, error) {
+	config := A2haHabitatAutoTfvars{}
+	err := json.Unmarshal([]byte(getConfigJsonString), &config)
+	if err != nil {
+		return "", err
+	}
+	authToken := config.HabSupHttpGatewayAuthToken
+
+	return authToken, nil
+}
+
+func (ss *Summary) getBEStatus(sshUtil SSHUtil, ip string, authToken, serviceType string) (BeStatusValue, error) {
+	service := ""
+	if serviceType == "Open Search" {
+		service = "automate-ha-opensearch"
+	} else if serviceType == "Postgresql" {
+		service = "automate-ha-postgresql"
+	}
+	args := []string{
+		"-s",
+		fmt.Sprintf("%s/services/%s/default", habUrl, service),
+		curlHeaderFlag,
+		fmt.Sprintf(curlAuthorization, authToken),
+		"-k",
+	}
+	defaultServiceDetails, err := executeCmd(sshUtil, "curl", args)
+	if err != nil {
+		return BeStatusValue{}, err
+	}
+	memeberId := defaultServiceDetails["sys"].(map[string]interface{})["member_id"]
+	serviceState := defaultServiceDetails["process"].(map[string]interface{})["state"].(string)
+	servicePid := fmt.Sprint(defaultServiceDetails["process"].(map[string]interface{})["pid"])
+	startingTime := defaultServiceDetails["process"].(map[string]interface{})["state_entered"].(float64)
+	startingTime = float64(nowFunc().UTC().Unix()) - startingTime
+	args = []string{
+		"-s",
+		fmt.Sprintf("%s/services/%s/default/health", habUrl, service),
+		curlHeaderFlag,
+		fmt.Sprintf(curlAuthorization, authToken),
+		"-k",
+	}
+	ServiceHealth, err := executeCmd(sshUtil, "curl", args)
+	if err != nil {
+		return BeStatusValue{}, err
+	}
+
+	health := fmt.Sprint(ServiceHealth["status"])
+
+	args = []string{
+		"-s",
+		fmt.Sprintf("%s/census", habUrl),
+		curlHeaderFlag,
+		fmt.Sprintf(curlAuthorization, authToken),
+		"-k",
+	}
+	censusData, err := executeCmd(sshUtil, "curl", args)
+	if err != nil {
+		return BeStatusValue{}, err
+	}
+
+	populationData := censusData["census_groups"].(map[string]interface{})[service+".default"].(map[string]interface{})["population"].(map[string]interface{})[fmt.Sprintf("%s", memeberId)]
+	getrole, ok := populationData.(map[string]interface{})
+	role := "Unknown"
```

Comment:
For the OS node, use `Node` for Role, instead of `Unknown`. Confirm once with @vivek-yadav